src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -818,11 +819,21 @@ ApplicationManagerImpl::driver_distraction_state() const {
   return driver_distraction_state_;
 }
 
+LockScreenDismissalEnabled
+ApplicationManagerImpl::lock_screen_dismissal_enabled() const {
+  return lock_screen_dismissal_enabled_;
+}
+
 void ApplicationManagerImpl::set_driver_distraction_state(
     const hmi_apis::Common_DriverDistractionState::eType state) {
   driver_distraction_state_ = state;
 }
 
+void ApplicationManagerImpl::set_lock_screen_dismissal_state(
```

Comment:
@IGapchuk setter and getter have different naming convention. Fix it

Reply:
@LuxoftAKutsan  Fixed in cd078272812e56dbc954d3d9319ac822f24b3093.